CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ Currently the versioning policy of this project follows [Semantic Versioning v2.
 ### Fixed
 - Do not report BC_UNCONFIRMED_CAST for Java 21's type switches when the switch instruction is TABLESWITCH ([#2782](https://github.com/spotbugs/spotbugs/issues/2782))
 - Do not throw exception when inspecting empty switch statements ([#2995](https://github.com/spotbugs/spotbugs/issues/2995))
+- Adjust priority since relaxed mode reports even `IGNORED_PRIORITY` ([#2994]https://github.com/spotbugs/spotbugs/issues/2994)
```

Comment:
It looks like there is one `(` missing for the correct formatting after the `]`. Can you please add it?

Reply:
Good catch @JuditKnoll 
I have made #3012 to fix other errors I've made